google/cloud/storage/client_notifications_test.cc

```diff
@@ -120,7 +119,7 @@ TEST_F(NotificationsTest, CreateNotification) {
             EXPECT_THAT(r.json_payload(), HasSubstr("test-object-prefix-"));
             EXPECT_THAT(r.json_payload(), HasSubstr("OBJECT_FINALIZE"));
 
-            return std::make_pair(Status(), expected);
+            return make_status_or( expected);
```

Comment:
Extra space after paren (et seq)?

Reply:
Fixed (here and elsewhere). I guess I trusted clang-format too much.